olive/packaging/sample_code/ONNXModel/cpp/README.md

```diff
@@ -1,8 +1,15 @@
 # Olive output instruction
 
 ## ONNXRuntime installation
+### Option 1: follow the installation instruction
 Please follow the instruction to install ONNX Runtime: https://onnxruntime.ai/docs/install/#cccwinml-installs
 
+### Option 2: install by local NuGet file
+You can find ONNXRuntime NuGet package file in `ONNXRuntime` folder. Install the local NuGet file by:
+```
+Install-Package <absolute-path-to-your-output-folder>\SampleCode\cpp\ONNXRuntime\<onnxruntime-package-name>.nupkg
```

Comment:
Is the command `Install-Package` workable for Linux also? I have no experiences on this. Just for double check.

Reply:
No this doesn't work for Linux. Not sure how people use nupkg package on Linux but yeah this should be a universal installation instruction. I'll update this to dotnet command instead.